docs/socialaccount/providers/google.rst

```diff
@@ -14,24 +14,42 @@ Create a google app to obtain a key and secret through the developer console.
 Google Developer Console
     https://console.developers.google.com/
 
-After you create a project you will have to create a "Client ID" and fill in
-some project details for the consent form that will be presented to the client.
-
-Under "APIs & auth" go to "Credentials" and create a new Client ID. Probably
-you will want a "Web application" Client ID. Provide your domain name or test
-domain name in "Authorized JavaScript origins". Finally fill in
-``http://127.0.0.1:8000/accounts/google/login/callback/`` in the
-"Authorized redirect URI" field. You can fill multiple URLs, one for each test
-domain. After creating the Client ID you will find all details for the Django
-configuration on this page.
-
-Users that login using the app will be presented a consent form. For this to
-work additional information is required. Under "APIs & auth" go to
-"Consent screen" and at least provide an email and product name.
+After you create a project, you will have to create an OAuth client ID and fill
+in some project details for the consent form that will be presented to the
+client:
+
+#. Under "APIs & Services", go to "Credentials", click "Create credentials" and
+   create a new "OAuth client ID". Probably you will want to choose "Web
+   application" as the application type. Provide your domain name or test
+   domain name in "Authorized JavaScript origins". Finally, fill in the
+   "Authorized redirect URIs" field with URLs like
+   ``http://example.com/accounts/google/login/callback/`` , replacing the
+   domain name with your domain name, or with ``127.0.0.1:8000`` for testing.
+   After creating the OAuth client ID, make a note of the client ID and the
+   client secret, as you will need it later.
+
+#. Users that log in using the app will be presented a consent form. For this
+   to work, additional information is required. Under "APIs & Services", go to
+   "OAuth consent screen" and at least provide an email address and a product
+   name.
 
 
 Django configuration
 ********************
+
+The Google provider requires this additional dependency to be installed::
+
+    pip install pyjwt[crypto]
```

Comment:
This is not correct -- `allauth.socialaccount` requires a `pip install django-allauth[socialaccount]`, and that already includes the above. So nothing additional is required for Google.

Reply:
Bah! I missed that. I deleted this text now.

Reply:
No problem, thanks for improving the docs!

Reply:
Thanks for saying thanks! I just spent a couple of hours improving documentation to pre-commit, and I submitted a pull request, and it got rejected within four minutes with a "no thanks". I regret spending around an hour or two working on it, trying to phrase it as clear as possible.